src/bip44.c

```diff
@@ -492,6 +546,11 @@ bool bip44_isPathReasonable(const bip44_path_t* pathSpec)
 	case PATH_MULTISIG_STAKING_KEY:
 		return bip44_hasReasonableAccount(pathSpec) && bip44_hasReasonableAddress(pathSpec);
 
+	case PATH_DREP_KEY:
+	case PATH_COMMITTEE_COLD_KEY:
+	case PATH_COMMITTEE_HOT_KEY:
+		return bip44_hasReasonableAccount(pathSpec);
```

Comment:
Nit: Maybe also `&& bip44_hasReasonableAddress(pathSpec)` so that this check works even if the `address_index == 0` condition is relaxed?